doc/api/errors.markdown

```diff
@@ -0,0 +1,470 @@
+# Errors
+
+<!--type=misc-->
+
+Errors generated by Node.js fall into two categories: JavaScript errors and system
+errors. All errors inherit from or are instances of JavaScript's [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)
+class and are guaranteed to provide *at least* the attributes available on that
+class.
+
+When an operation is not permitted due to language-syntax or
+language-runtime-level reasons, a **JavaScript error** is generated and thrown
+as an **exception**. If an operation is not allowed due to system-level
+restrictions, a **system error** is generated. Client code is then given the
+opportunity to **intercept** this error based on how the API **propagates** it.
+
+The style of API called determines how generated errors are handed back, or
+**propagated**, to client code, which in turn informs how the client may **intercept**
+the error. Exceptions can be intercepted using the `try / catch` construct;
+other propagation strategies are covered [below](#errors_error_propagation_and_interception).
+
+## JavaScript Errors
+
+<!--type=misc-->
+
+JavaScript errors typically denote that an API is being used incorrectly, or that
+there is a problem with the program as written.
+
+### Class: Error
+
+<!--type=class-->
+
+A general error object. Unlike other error objects, `Error` instances do not
+denote any specific circumstance of why the error occurred. Errors capture a
+"stack trace" detailing the point in the program at which they were
+instantiated, and may provide a description of the error.
+
+**Note**: Node will generate this class of error to encapsulate system
+errors as well as plain JavaScript errors.
+
+#### new Error(message)
+
+Instantiates a new Error object and sets its `.message` property to the provided
+message. Its `.stack` will represent the point in the program at which `new Error`
+was called. Stack traces are subject to [V8's stack trace API](https://code.google.com/p/v8/wiki/JavaScriptStackTraceApi). 
```

Comment:
If I follow this link, it says that the page has moved to https://code.google.com/p/v8-wiki/wiki/JavaScriptStackTraceApi